mcimx6q-sdb.coffee

```diff
@@ -0,0 +1,34 @@
+deviceTypesCommon = require 'resin-device-types/common'
```

Comment:
By convention the filename is always <YOCTO_MACHINE>.coffee . This is necessary for barys to find the right json file easily.